examples/webrtc-client-example/components/client.details/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { ChangeEvent } from 'react'
+import styles from './index.module.css'
+import { keyOptionList } from '../../library/constants'
+
+export interface ClientDetailsProps {
+  streamId?: string
+  setStreamId?: Function
+  apiKey: string
+  setApiKey: Function
+  apiKeyType: string
+  setApiKeyType: Function
+  showChannel: boolean
+}
```

Comment:
Fine for the demo app, but we generally follow alphabetical order while declaring interface attributes.